python-sanic/README.md

```diff
@@ -0,0 +1,33 @@
+
+# Quickstart for debugging Sanic
+
+A sample application for debugging Python Sanic apps using Rookout.
+
+Before following this guide we recommend reading the basic [Python + Rookout](https://docs.rookout.com/docs/sdk-setup.html) guide.
+
+This sample may be out of date. If you face any issues, please reach out to mailto:support@rookout.com and let us know.
+
+## Running Sanic Server with Rookout
+
+1. *Clone and install dependencies*:
```

Comment:
write about the diff between with worker

Reply:
I wrote an explanation about the workers in the `Rookout Integration Explained` section 
If its not enough please tell me :)